### DISCUSSION TITLE
Multiple campuses; one company

### DISCUSSION THREAD

**Discussion by bodybraille** [snipe/snipe-it#9796] [Q&A]
You can create containers that separate devices by model, cpu, memory, etc; but can you separate devices by their name and/or location

Where in the documentation can I find configuring assets to filter into a category/container by name or location? is it possible?

All the devices that belong to campus A will automatically get sorted to the campus A container based on computer name/location. Campus B has it's own naming convention, so on and so forth.

Right now I have thousands of devices with different names that belong to six different campuses.

**Comment by bveina** [comment]
not sure if this is what you are looking for, but i routinely filter my assets by location by going to list all assets then typing the location name into the search field. this gives me a list of every item found in that room.
